tests/testenv/mod.rs

```diff
@@ -161,6 +161,14 @@ impl TestEnv {
         }
     }
 
+    pub fn set_environment_variable(&self, key: &str, value: &str) {
```

Comment:
These functions might not work well when multiple tests are run in parallel. Right now, it won't be an issue, because only a single test is using these functions. But if there are multiple tests, then one test might set this environment variable in one thread, then another test will overwrite that in another thread before the first test has a chance to execute `fd`. It would probably be better to set the enviornment when we spawn the `fd` process directly.

Reply:
Actually... I think it could be a problem currently, because the test that uses FD_CONFIG_FILE could run at the same time as another test, and result in that test starting with the config file set, and inheriting the options from the config file.